Game/RainMeadow.PlayerHooks.cs

```diff
@@ -2057,10 +2057,13 @@ private bool Player_CanIPickThisUp(On.Player.orig_CanIPickThisUp orig, Player se
                     }
                     if (obj is Player pl)
                     {
-                        if (pl.Stunned || pl.dead)
-                        {
+                        if (pl.dead || pl.Stunned) {
+                            if (pl.dead & !arena.piggyBack)// no grabbing period
+                            {
+                                return false;                  
+                            };
                             return orig(self, obj);
-                        };
+                        }
```

Comment:
Use logical AND instead of bitwise AND.
```suggestion
                        if (pl.dead || pl.Stunned) {
                            if (pl.dead && !arena.piggyBack)// no grabbing period
                            {
                                return false;                  
                            };
return orig(self, obj);
                        }
```

Alternatively


```suggestion
                        if (!arena.piggyBack) return false;
```